roles/ceph-nfs/tasks/pre_requisite_container.yml

```diff
@@ -39,5 +39,26 @@
       when:
         - cephx | bool
         - item.item.copy_key | bool
+      no_log: true
+
   when: groups.get(mon_group_name, []) | length > 0
 
+- name: dbus related tasks
+  block:
+    - name: get file
+      command: "{{ container_binary }} run --rm --entrypoint=cat {{ ceph_docker_registry + '/' + ceph_docker_image + ':' + ceph_docker_image_tag }} /etc/dbus-1/system.d/org.ganesha.nfsd.conf"
+      register: dbus_ganesha_file
+      run_once: true
+      changed_when: false
+
+    - name: create dbus service file
+      copy:
+        content: "{{ dbus_ganesha_file.stdout }}"
+        dest: /etc/dbus-1/system.d/org.ganesha.nfsd.conf
+        owner: "root"
+        group: "root"
+        mode: "0644"
+
+    - name: reload dbus configuration
+      command: "killall -SIGHUP dbus-daemon"
+  when: ceph_nfs_dynamic_exports | bool
```

Comment:
this shouldn't be here. As @guits said, you grabbed an extra commit that you shouldn't have.

Please try again :)

Reply:
Sure let me retry with a new pull request.

Reply:
You don't have to keep creating new pull requests 😄
If you just fix it in your fork (even reclone, and do changes again) and push it'll show up in here. You can also check out 'git reflog'
I think PRs records stay so won't look too nice in ceph-ansible repo

Reply:
Sure , thanks for the advice . Since this was my first time creating a verified pull request along with squashing of commits. Learnt a lot.

Reply:
Created a new pull request here #6538